include/boost/math/distributions/von_mises.hpp

```diff
@@ -0,0 +1,842 @@
+//  Copyright John Maddock 2006, 2007.
+//  Copyright Paul A. Bristow 2006, 2007.
+
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_STATS_VON_MISES_HPP
+#define BOOST_STATS_VON_MISES_HPP
```

Comment:
I think you've picked up an artefact of an old directory structure from other files. I'd change this to `BOOST_MATH_DISTRIBUTIONS_VON_MISES_HPP`.

Reply:
Agree.